### DISCUSSION TITLE
aw-watcher-web for Edge browser

### DISCUSSION THREAD

**Discussion by goodhalf** [ActivityWatch/activitywatch#943] [Ideas]
I switched to Edge about 2 years ago and have been using it since. I'm just now getting into the idea of time tracking and would super appreciate if this was an extension for Edge. This project seems really awesome.

**Comment by BelKed** [comment]
There is actually an Edge extension that you can download from the Google Chrome Store:
https://chrome.google.com/webstore/detail/activitywatch-web-watcher/nglaklhklhcoonedhgnpgddginnjdadi

I've just tried to install it and it worked fine for me :)

---

<sup>By the way, this extension should work with any Chrome-based web browser ;)</sup>
